.gitignore

```diff
@@ -1,6 +1,7 @@
 **/node_modules/**
 **/generated/**
-
+target/
+export/
```

Comment:
could you move these two to a .gitignore in your project's directory (instead of root .gitignore)

Reply:
Done